portfolio/src/main/webapp/index.html

```diff
@@ -8,19 +8,18 @@
   </head>
   
   <body>
-      <!--need to fix navbar for when resize window-->
       <nav class="navbar">
         <div>
             <ul>
                 <table>
                     <tr>
-                        <td> <li><a href="#education">Education</a></li> </td>
-                        <td> <li><a href="#work">Work</a></li></td>
-                        <td><li><a href="#projects">Projects</a></li></td>
-                        <td><li><a href="#service">Community Service</a></li></td>
-                        <td><li><a href="#interets">Interets</a></li></td>
-                        <td><li><a href="#family">Family</a></li></td>
-                        <td><li><a href="#contact">Contact Info</a></li></td>
+                        <td><li class = "nav"><a href="#education">Education</a></li> </td>
```

Comment:
So, this entire table should actually be replaced with a `ul`. Let's merge this to master and do a follow up in a later PR. I can do a demo maybe at our 1:1 or team meeting.

Tables are meant to hold tabular data and are fairly rarely used for other purposes. Table-based layouts used to be extremely popular but there are a number of problems that have been solved in recent years such that using tables for this purpose has fallen out of favor.